infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/receive/LogReplicationMetadataManager.java

```diff
@@ -91,7 +91,7 @@ public LogReplicationMetadataManager(CorfuRuntime rt, long topologyConfigId, Str
                     ReplicationEventKey.class,
                     ReplicationEvent.class,
                     null,
-                    TableOptions.builder().build());
```

Comment:
The tests failing are because in this file Line 87 you should fetch table Option from proto schema.

Reply:
done. thanks so much!